Scripts/flow.php

```diff
@@ -20,6 +20,10 @@
 }
 
 if (isset($argv[1]) && ($argv[1] === 'typo3.flow:core:setfilepermissions' || $argv[1] === 'flow:core:setfilepermissions' || $argv[1] === 'core:setfilepermissions')) {
+    $perms = decoct(fileperms(__DIR__. '/setfilepermissions.sh') & 0777);
+        if ($perms != '700'){
```

Comment:
seems like an indention error.
But apart from that: Instead of checking for 0700 wouldn't an `is_executable()` check work and be a bit less strict?